GSA/digitalgov.gov
# Sidebar updates - Section Removal and Spacing Clean Up

## Summary

This Pr will update side bar and remove `get started building` section and fix spacing for authors section... this is also a cont. from PR - https://github.com/GSA/digitalgov.gov/pull/6869#top

### Preview

[Resources Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-sidebar-update/resources/21st-century-integrated-digital-experience-act/?dg)
[Event Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-sidebar-update/event/2023/10/19/uswds-monthly-call-october-2023/)
[News Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-sidebar-update/2023/10/31/who-is-controlling-your-control-system/)

Example (before/after)
<img width="312" alt="Screenshot 2023-11-08 at 4 14 10 PM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/33f7364e-3532-4cca-b279-e9ffd14c3968"><img width="280" alt="Screenshot 2023-11-08 at 4 14 31 PM" src="https://github.com/GSA/digitalgov.gov/assets/88721460/48002b9e-63c2-42ad-8c03-472a3316c9bc">





### Solution

1. removed section for `get started building` 
2. updated style sheets for the authors section to remove unnecessary margins 

### How To Test

1. Visit preview links above (or any pages with sidebar).
2. Ensure sections are consistently styled and spaced.
3. Ensure there are no regressions.
4. Verify `Get Started Building` section has been removed 

---

### Dev Checklist
